packages/webapp/components/posts/PostWidgets.tsx

```diff
@@ -49,13 +36,10 @@ export function PostWidgets({ postById }: PostWidgetsProps): ReactElement {
 
   return (
     <PageSidebar>
-      {postById && <ShareBar post={postById.post} />}
+      <ShareBar post={post} />
```

Comment:
The check for `postById` was there on purpose because it's being fetched on load.
If this is not defined this might cause an error.
```suggestion
      {post && <ShareBar post={post} />}
```

Reply:
From the line of code below, we return whenever the `postById` or `postById.post` is `null` or `undefined`, so the code below this point (including the rendering part of this component) should be safe. Would we use this component for other parts as well?

https://github.com/dailydotdev/apps/blob/c7995b1e4b7071464e91aa6cd584466b184758ce/packages/webapp/pages/posts/%5Bid%5D.tsx#L201-L203

Reply:
From the line of code below, we return whenever the `postById` or `postById.post` is `null` or `undefined`, so the code below this point (including the rendering part of this component) should be safe. Would we use this component for other parts as well?

https://github.com/dailydotdev/apps/blob/c7995b1e4b7071464e91aa6cd584466b184758ce/packages/webapp/pages/posts/%5Bid%5D.tsx#L201-L203

Reply:
You're right! I missed that